projects/plugins/super-cache/wp-cache.php

```diff
@@ -133,6 +133,15 @@ function wp_cache_set_home() {
 }
 add_action( 'template_redirect', 'wp_cache_set_home' );
 
+function wpsc_enqueue_styles() {
+	wp_enqueue_style(
+		'wpsc_styles',
+		plugins_url( 'styling/dashboard.css', __FILE__ ),
+		array(),
+		filemtime( plugin_dir_path( __FILE__ ) . 'styling/dashboard.css' )
+	);
+}
+add_action( 'admin_enqueue_scripts', 'wpsc_enqueue_styles' );
```

Comment:
The styles are leaking into wp-admin. Checking for $_GET['page'] === 'wpsupercache' before adding the action should fix it.

Reply:
Fixed in ca3d9f9 👍